paper.tex

```diff
@@ -628,34 +628,36 @@ \subsection{Outsourcing}
 Another possibility for the realisation of local RSE Service providers is by forming a spin-off and pooling the RSE Skills into an external company, which has benefits but also drawbacks. [\#SuccessStory Outsourcing]
 Among the most obvious benefits is that this enables the creation of contracts outside of the WissZeitVG.\@
 This also widens the customer base of the RSE unit since the newly founded company may obtain contracts from industry.
-If this company is university backed/branded this enables another possibility for a university to interact with the local society.
+If this company is backed/branded by the institution, this enables another possibility for an institution to interact with the local society.
 But there are drawbacks.
-Since the company is now a university external entity the Vergabe-Richtlinien have to be fulfilled, which could \eg{} mean to publicly invite tenders in order to have a competitive procedure.
+Since the company is now an entity external to the institution, the Vergabe-Richtlinien have to be fulfilled, which could \eg{} mean to publicly invite tenders in order to have a competitive procedure.
 This also points to the fact that an external company has to be a mostly profitable entity (partly this can be softened by founding a non-for-profit entity).
 Moreover, during the outsourcing contract, there has to be a coordinator at both sides and the flow of information from the academic institution to the contracted company has to be established.
 These are some examples of additional administrative overhead due to the interaction with external partners.
 Certain domains will have issues with ensuring data privacy when working with this external company.
-If forming a public company, it is hoped that parts of these issues can be legally alleviated by forming respective framework agreements between a university and the company.
+Some of these issues might be legally alleviated by forming framework agreements between an institution and the company.
 It is of particular importance to agree on requirements and handover criteria, including quality assurance and license specifications.
 On top of these drawbacks, there are soft factors, like whether an external company is accepted by scientists.
 
 \subsection{Staff Acquisition/People}
 
-This leaves us with the question of, assuming we have centralised RSE units, where do we get the employees?
-Being an RSE should be a career worth aspiring to as any other profession and hence we have to additionally create education structures that generate a pool of people where central RSE units can be populated from.
-This is a topic that will be considered in a separate paper.
+RSE units need to be staffed, but where do potential employees come from?
+Educational structures that generate a pool of people from which central RSE units can be populated from need to be set up.
```

Comment:
This sounds quite inhumane and very corporate, maybe 
```suggestion
So far, researchers accidentally find themselves in the role of an RSE because they pursued software development as part of their research.
A more deliberate approach with specific RSE education can train people in sufficient numbers for central RSE units.
```